libs/sdk-ui-dashboard/src/presentation/topBar/buttonBar/button/selectors.ts

```diff
@@ -85,9 +85,9 @@ export function selectCanSaveDashboard(state: DashboardState) {
 export const selectHasPermissionsForShare = createSelector(
     selectEnableAnalyticalDashboardPermissions,
     selectSupportsAccessControlCapability,
-    selectCanManageACL,
-    (arePermissionsEnabled, supportsAccessControl, canManageACL) =>
-        arePermissionsEnabled && supportsAccessControl && canManageACL,
+    selectCanShareDashboardPermission,
```

Comment:
Moving from bear-specific workspace-level permission to backend agnostic dashboard permissions.
They use the same logic on Bear - see `canShareDashboard: workspacePermissions.canManageACL`